src/math/basic/legendrePolynomials.cpp

```diff
@@ -706,6 +706,38 @@ double computeLegendrePolynomialExplicit( const int degree,
         }
         }
         break;
+    case 5:
+        switch( order )
+        {
+        case 0:
+            return ( 63.0 * polynomialParameter * polynomialParameter
```

Comment:
Have you considered using `std::pow(polynomialParameter, 5)` here instead? It might offer a more concise and readable solution.